src/main/obsRecorder.ts

```diff
@@ -382,11 +382,16 @@ const assertNextSignal = async (value: string) => {
   console.debug("[OBS] Asserted OBS signal:", value);
 }
 
+const getObsLastRecording = (): string => {
```

Comment:
Worth a comment here to specify exactly what is returned? e.g. Relative path / absolute path as a string?

Reply:
Maybe even worth validating with path.resolve?

Reply:
I knew you were going to comment on it! :D I thought it was self-explanatory enough, but I'll add a comment :D